### ISSUE TITLE
sshuttle to host with large routing table results in AssertionError

### ISSUE THREAD

**Issue by krokodilerian** [sshuttle/sshuttle#306]
Trying to sshuttle to a node with large (full bgp) routing table results in AssertionError:

```
vasil@nymphadora:~$ sshuttle -r xxx@yyyyyy 172.31.1.0/24      
client: Connected.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "assembler.py", line 38, in <module>
  File "sshuttle.server", line 310, in main
  File "sshuttle.ssnet", line 379, in send
AssertionError
client: fatal: server died with error code 1
```

From a workaround I found on the internet (basically having `list_routes()` return an empty list) I've found that the issue is that `mux.send()` can't send messages larger than 64K, and the routing table is definitely bigger.

I'm willing to write a patch for this, the question is what is the proper direction?
- send just the beginning of the routing table
- if not needed, do not request the routing table
- send the routing table in chunks (which might require an update in the internal protocol)
- add support for larger messages in the protocol (no idea if it's useful)

**Comment by brianmay** [comment]
I think this routing table is only ever needed if using `--auto-nets` so maybe we should only ever transfer it on request by the client?

Apart from that, what is the simplest solution?

**Comment by krokodilerian** [comment]
I did a simple PR with just passing --auto-nets (https://github.com/sshuttle/sshuttle/pull/311). It was simple enough, and I think I can just limit the routing table to fit in 64KB, but that seems pretty hacky to me. OTOH, no idea who would need more routes than that.

**Comment by brianmay** [comment]
Fixed by #311.
